src/network/realtime.js

```diff
@@ -593,7 +593,7 @@ createNameSpace("realityEditor.network.realtime");
 
         if (typeof objectSocketCache[objectKey] === 'undefined') {
             var object = realityEditor.getObject(objectKey);
-            var serverIP = object.ip;
+            var serverIP = "localhost";
```

Comment:
All of this server ip/url logic needs a more solid solution than making everything localhost. It's something I can look at as part of making a more complete server-userinterface combination

Reply:
This is a part of https://realitylab.atlassian.net/browse/POPUP-228. I can dive into it, but it might be better for you to address this since I have little experience with this part of the code.